### ISSUE TITLE
What if my dataset only contains 300 samples for training using this model?

### ISSUE THREAD

**Issue by Frank-LIU-520** [ChakradharG/PeptideBERT#8]
Do you have any suggestions to tuning this model for better convergence?
How to set it into regression model?
Now my datasets contain heavy chain and light chain, what should I do since it has two sequences if I want to use your model.

**Comment by ChakradharG** [comment]
1. I suggest you play around with the hyperparameters until you get a reasonable convergence on your specific dataset. A good place to start might be to tweak the architecture of the model depending on the length and complexity of your peptide sequences.
2. To use it as a regression model, remove the final sigmoid layer in network architecture (`network.py`). For training, you'd need to change several things: loss function, scheduler mode (min), and metrics being computed during epochs.
3. You can train 2 instances of PeptideBert, one for heavy chain and one for light chain, and then train another MLP that combines the output of these 2 models to give you a single prediction. Alternatively, you could simply concatenate both chains into 1 sequence, but I don't expect this method to be on par with the former method.

**Comment by Frank-LIU-520** [comment]
I would like to do just MLM for in-house sequences. What should I do?

**Comment by ChakradharG** [comment]
You should load the model checkpoint from this repository and follow the steps outlined in HuggingFace's documentation for MLM.
